posts/inside-rust/2023-04-12-trademark-policy-draft-feedback.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "A note on the Trademark Policy Draft"
+author: Ryan Levick, Jane Losare-Lusby, Tyler Mandry, Mark Rousskov, Josh Stone, and Josh Triplett
+team: The Foundation Project Directors and informal Trademark Working Group Members
+---
+
+# A note on the Trademark Policy Draft
+
+The Rust Foundation Project Directors have been working with the informal Trademark Working Group and the Foundation staff to draft an updated policy and FAQ for the Rust trademark. We'd like to provide some context around this work and address the community response to the [feedback form] for the recently circulated draft.
```

Comment:
Is it worth calling out that this process has been going on for several months? (And is not solely in reaction to posting of the draft)

Reply:
I concerned that without an explicit call out, it could be interpreted as a reaction to the current controversy, rather than something that has been ongoing for quite some time.

Reply:
Good point,

```suggestion
For the past eight months, the Rust Foundation Project Directors have been working with the informal Trademark Working Group and the Foundation staff to draft an updated policy and FAQ for the Rust trademark. We'd like to provide some context around this work and address the community response to the [feedback form] for the recently circulated draft.
```